lib/chef/knife/ec2_server_create.rb

```diff
@@ -927,8 +927,10 @@ def eip_scope
       end
 
       def ssl_config_user_data
-<<-EOH
 
+<<-EOH
+net user /add #{locate_config_value(:winrm_user).delete('.\\')} #{windows_password};
+net localgroup Administrators /add #{locate_config_value(:winrm_user).delete('.\\')};
```

Comment:
I suppose there isn't a circumstance where someone will be using this feature to bootstrap a server in a domain on ec2 over ssl, but it does seem like we should only being doing this step if we're given a local user.